index.html

```diff
@@ -1,238 +1,276 @@
+            <img src="Images/SnapshootPortfolio1.png" alt="SnapshootPortfolio1" class="img3">
+
+        </div>
+
+
+        <div class="project">
+            <img src="Images/SnapshootPortfolio2.png" alt="SnapshootPortfolio2" class="img1">
+            <img src="./Images/SnapshootPortfolio-desktop3.png" alt="SnapshootPortfolio" class="img2">
+            <img src="Images/SnapshootPortfolio2.png" alt="SnapshootPortfolio2" class="img3">
+
+            <div>
+                <h2>Tonic</h2>
+
+                <ol>
+                    <li>CANOPY</li>
+                    <li><img src="Images/Counter.png" alt="Counter"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="Images/Counter.png" alt="Counter"></li>
+                    <li>2015</li>
+                </ol>
+
+                <p>A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScipt</li>
+                </ul>
+                <button type="button" class="button"> See Project</button>
+            </div>
+        </div>
+
+
+        <div class="project">
+            <img src="Images/SnapshootPortfolio3.png" alt="SnapshootPortfolio3" class="img1">
+            <div>
+                <h2>Multi-Post <br class="br">Stories</h2>
+
+                <ol>
+                    <li>CANOPY</li>
+                    <li><img src="Images/Counter.png" alt="counter"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="Images/Counter.png" alt="counter"></li>
+                    <li>2015</li>
+                </ol>
+
+                <p>A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScipt</li>
+                </ul>
+
+                <button type="button" class="button"> See Project</button>
+            </div>
+            <img src="./Images/SnapshootPortfolio-desktop4.png" alt="SnapshootPortfolio" class="img2">
+            <img src="Images/SnapshootPortfolio3.png" alt="SnapshootPortfolio3" class="img3">
+
+        </div>
+    </section>
 
-         
-             <li>
-                 <h2>Languages <i class="fas fa-angle-down"></i></h2>
-                 <ul class="tech-skills">
-                    <li>
-                       <div class="tech-skills-detail">
-                         <img src="Images/Ellipse1.png" alt="Javascript">
-                         <p>Javascript</p>
-                      </div>
-                    </li>
 
-                 <li>
-                   <div class="tech-skills-detail">
-                      <img src="Images/Ellipse2.png" alt="HTML">
-                      <p>HTML</p>
-                    </div>
-                 </li>
-            
+
+    <section class="about" id="about">
```

Comment:
Making the about section container a Flexbox container will help you layout its contents as you like in different screen sizes (mobile and desktop)